x-pack/plugins/siem/server/lib/hosts/elasticsearch_adapter.test.ts

```diff
@@ -68,7 +68,8 @@ describe('hosts elasticsearch_adapter', () => {
     test('it formats a host with a source of name correctly', () => {
       const fields: ReadonlyArray<string> = ['host.name'];
       const data = formatHostEdgesData(fields, buckets);
-      const expected: HostsEdges = {
+      // TODO: Re-type formatHostEdgesData since it does not convert things to string arrays and re-use it here
```

Comment:
with my PR coming I do not think that we need to do that

Reply:
Hosts will have its own type and I think that's better since the query is all about aggregations and we have control about it